travis/setup_dependencies_common.sh

```diff
@@ -230,6 +230,13 @@ fi
 
 # ASTROPY
 if [[ ! -z $ASTROPY_VERSION ]]; then
+    # force uninstall hypothesis if it's silentry installed as an upstream
+    # dependency as the astropy <2.0.3 machinery is incompatible with
+    # it. But keep it if it's installed as an explicit dependency in
```

Comment:
Maybe 'But keep it if it's installed' -> 'But we do this so that here so that it will be re-installed as an explicit dependency...'